src/common/indexer/elastic/elastic.indexer.helper.ts

```diff
@@ -127,14 +135,38 @@ export class ElasticIndexerHelper {
     }
 
     if (filter.excludeMetaESDT === true) {
-      elasticQuery = elasticQuery.withMustMultiShouldCondition([NftType.NonFungibleESDT, NftType.SemiFungibleESDT], type => QueryType.Match('type', type));
+      elasticQuery = elasticQuery.withMustMultiShouldCondition([
+        NftType.NonFungibleESDT,
```

Comment:
we could have defined some subgroups, something like:
```
export const semiFungibleGroup = [
  NftType.SemiFungibleESDT,
  NftType.DynamicSemiFungibleESDT,
];
export const nonFungibleGroup = [
  NftType.NonFungibleESDT,
  NftType.NonFungibleESDTv2,
  NftType.DynamicNonFungibleESDT,
];
```

This part of the code can become:
```
semiFungibleGroup.concat(dynamicGroup)
```
which is also future proof

Reply:
very good idea, implemented it for now as a member of the class itself